src/components/form-container.js

```diff
@@ -0,0 +1,180 @@
+import React from "react"
+import Input from './input'
+import Summary from './summary'
+import Result from './result'
+
+const uuid = require('uuid')
+
+class FormContainer extends React.Component {
+	state = {
+		fullName: '',
+		attendingCeremony: false,
+		attendingBanquet: false,
```

Comment:
Continuing from the other comments, you could structure your state to hold the values of your guest form, and then the `guests` will contain everyone, including the leader which happens to be the first guest. Since the array is an ordered collection, the first guest will always appear as the first guest.